Documentation/tag-set.md

```diff
@@ -4,30 +4,42 @@ This tag stores a value into a variable for later use.
 
 ## Node Information
 
-| Name      | Description                      |
-|-----------|----------------------------------|
-| Tag Name  | `set`                            |
-| End Tag   | `endset`                         |
-| Rendering | Immediately; no output           |
+| Name      | Description                                                            |
+|-----------|------------------------------------------------------------------------|
+| Tag Name  | `set`                                                                  |
+| End Tag   | `endset`  or N/A (if you're creating an alias of an existing variable) |
+| Rendering | Immediately; no output                                                 |
 
-| Parameter | Description                                 | 
-|-----------|---------------------------------------------|
-| Name      | The name of the variable you want to store. |
+| Parameter  | Description                                 | 
+|------------|---------------------------------------------|
+| Name       | The name of the variable you want to store. |
+| Expression | (Optional) The value to be stored.          |
 
-_Example:_ `{% set myVar %}hello{% endset %}`
+The parameters and tags of this node depend on which mode you want to use:
+
+- Use `{% set myVar %}...{% endset %}` to render and store everything between the start and end tag into the variable.
+- Use `{% set myVar someOtherVar.prop1.prop2 #}` to evaluate and store an expression's result into the variable.
```

Comment:
I've seen you twice (here and in the PR description) tell that this ends with `#}` not `%}`. Is this intentional or a double typo?

Reply:
I have no idea how that got in there 😆 It's in the PR description because I copy/pasted it.